frontend/app-development/App.test.tsx

```diff
@@ -22,13 +31,14 @@ describe('App', () => {
         } as IUserState,
       },
     });
-
+    await screen.findByTestId('app-content-wrapper');
```

Comment:
Det er ikke lett å se hva som testes her. Det er bedre å buke `expect`, som på linjen under.
```suggestion
    expect(screen.findByTestId('app-content-wrapper')).toBeInTheDocument();
```
Har du for øvrig vurdert andre selektorer enn test-id?

Reply:
Brukes isteden for waitfor... fordi henting av repostatus er asynkron.